BioForm/profiles/management/commands/setuserinactive.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+from profiles.models import Account
+from datetime import date
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        count = 0
+        active_users = Account.objects.filter(is_active=True)
```

Comment:
You can apply the condition in the filter to get only users who have not signed in for the last 10 days.

Reply:
Change made